R/rater_functions.R

```diff
@@ -61,7 +69,10 @@ meanKappa <- function( dat , type = c("Cohen", "BrennanPrediger"), weight = "unw
   # weight.mean ... = T, if agreement is weighted by number of rater subjects,
   #            = F, if it is averaged among all rater pairs
   dat  <- eatTools::makeDataFrame(dat)
+  checkmate::assert_character(weight, len = 1)
+  checkmate::assert_logical(weight.mean, len = 1)
   type <- match.arg(type)
+
```

Comment:
I understood line 74 as: it gives `type` a DEFAULT value (Cohen).
should check type for chr and len = 1; or as a subset of c(Cohen, BrennanPrediger)? or just leave it as it is?

Reply:
yes, please as a subset of c("Cohen", "BrennanPrediger")